crates/driver/README.md

```diff
@@ -0,0 +1,47 @@
+# Driver
+
+The purpose of the driver is to make it easier to author, run, and maintain solvers.
+
+All solvers do the following:
+
+- collect some onchain liquidity (optional),
```

Comment:
Funny to start a list of things **all solvers do** with something optional. 😄 
I think the wording can stay as is; just found it funny.